debian/changelog

```diff
@@ -0,0 +1,5 @@
+libjsonv (1.0.0-pre0-1) unstable; urgency=low
```

Comment:
As a general statement, can you move all the things in the `debian` folder to `installer/debian`?